tbx/project_styleguide/templates/patterns/pages/work/work_page_base.md

```diff
@@ -0,0 +1,3 @@
+# Work page base template
+
+This is the base template for the historical work page and the new work page - there is no config set up for this in the pattern library - see the child pages instead.
```

Comment:
**Praise**: 

1. for adding documentation to explain what's going on here.
2. for abstracting common elements into a base template, thus avoiding code duplication and making future maintenance easier